extensions/core/deployment/src/main/java/io/quarkiverse/cxf/deployment/CxfBuildTimeConfig.java

```diff
@@ -445,4 +445,116 @@ default void validate(String prefix) {
         }
     }
 
+    /**
+     * Parameters for fully customizable algorithm suite.
+     */
+    @WithName("customizedAlgorithmSuite")
```

Comment:
So this whole alg suite config should be used mostly for fips compliance, right? Then I'd be for the following structure of the configuration:

```
quarkus.cxf.security.fips.enabled = [true/false] # enables the custome alg. suite
quarkus.cxf.security.fips.algorithm-suite.digest-algorithm = ...
quarkus.cxf.security.fips.algorithm-suite.encryption-algorithm = ...
...
```

WDYT?

Reply:
I originally created it with fips property, but then I asked @ffang about his opinion. I might have understood or asked wrongly.

The advice was:
```
I'd use some name like customerizedPolicy and make all parameters configurable, it should be a general solution, so let the end users control everything
```

Reply:
Yes, I'm thinking of more general than just FIPS context. The underlying specification may stop evolving,  but if users can configure all parameters of the AlgorithmSuite to avoid using some cracked/out-dated security algo, this would be awesome. And this flexibility could not only fit FIPS scenairo.